### PR TITLE
Adding Battleball.

### PR DESCRIPTION
**Pull request by Quinncuatro**
Submitting this PR means you have:

- [x] searched the [readme](https://github.com/edm00se/awesome-board-games/blob/master/readme.md) file and searched the [open pull requests](https://github.com/edm00se/awesome-board-games/pulls) (no duplicates please)
- [x] and will follow [the desired format](https://github.com/edm00se/awesome-board-games/blob/master/formatting.md)
- [x] used image links that resolve correctly, are appropriately sized, and are served over HTTPS

Once your PR is created, it will trigger a build a GitHub Action (Awesome-Lint). This includes a format linting process, so if there are any issues with the build, please address them.

Do you have further questions? Check the [contribution guidelines](https://github.com/edm00se/awesome-board-games/blob/master/contributing.md) or [open an issue](https://github.com/edm00se/awesome-board-games/issues/new).

When you're ready, check off the above acknowledgements and continue with your Pull Request... and thanks!

### CODE REVIEW

**Review by edm00se — CHANGES_REQUESTED** [review]
The image is too large. You can see in the file path that it's a 900x600 pixel. The target size is ~250px X 250px. If it's close, the rest looks good.

**Comment by Quinncuatro** [comment]
Updated to include the smallest hosted image I could find.

Also, any chance this PR could have a "hacktoberfest-accepted" label added to it?

File: readme.md [hunk 1]
```diff
@@ -159,6 +159,16 @@
 | ------: | -------: | ---: |
 |       2 |        8 |  10m |
 
+### [Battleball](https://boardgamegeek.com/boardgame/6795/battleball)
+
+> Battleball is a football board game but with teams made of players with physical and mechanical enhancements - which has made the sport more intense and more deadly. Plan a strategy and do your best to score two touchdowns without having all of your players be destroyed!
+
+![Battleball Image](https://i.ebayimg.com/images/g/xFsAAOSwOMpd6G8M/s-l400.jpg)
```
Comment (edm00se) [inline readme.md:166]:
```suggestion
![Battleball Image](https://cf.geekdo-images.com/itemrep/img/_ybsCFVKunSnqyGPCiuXbT2kuDo=/fit-in/246x300/filters:strip_icc()/pic227687.jpg)
```

Like I mentioned in the first commit, there's a specified image size for the cdn path; finding the correct pre-generated image source means there's a more size appropriate version available to use.

File: readme.md [hunk 2]
```diff
@@ -414,6 +424,16 @@
 | ------: | -------: | -----: |
 |   2 - 4 |       13 | 30-45m |
 
+### [Heroscape](https://boardgamegeek.com/boardgame/11170/heroscape-master-set-rise-valkyrie)
+
+> Join the battle of all time! Draft your army from troops of all different eras, species, and battle styles then fight on a battlefield created by you and your friends. Work together to accomplish a scenario laid out in the included rules or duke it out amongst yourselves to see who's the greater battlefield commander.
+
+![Heroscape Master Set: Rise of the Valkyrie](https://cdn.shopify.com/s/files/1/1031/9405/products/1_13261260-646a-4892-a1ab-737aa730c902.jpeg?v=1589744301)
```
Comment (edm00se) [inline readme.md:431]:
```suggestion
![Heroscape Master Set: Rise of the Valkyrie](https://cf.geekdo-images.com/itemrep/img/QxW14xwSXjhYU8h_KlvG09nbBd0=/fit-in/246x300/filters:strip_icc()/pic244662.jpg)
```

Similar story to this one. Normally I'd like people to keep PRs for games separate, but it's fine.

**Review by edm00se — CHANGES_REQUESTED** [review]
If you can update those links, this should be good to go. I found a couple images that should be perfect and added them as suggested changes, which should make things easy.

File: readme.md (hunk 1 above)
Comment (edm00se) [reply readme.md]: This looks good.

File: readme.md (hunk 2 above)
Comment (edm00se) [reply readme.md:431]: This image hasn’t changed and its current resolution is still a bit over size (400x300), with a fair amount of white space. The suggested code change image source is 246x125 and has no extra white space in the image.

**Review by edm00se — APPROVED** [review]
Looks good, thanks for contributing.

**Pull request merged**